README.md

```diff
@@ -253,38 +486,7 @@ PARAMETER | REQUIRED | DESCRIPTION | TYPE | CONTAINS
 
 No Output
 
-## action: 'community lookup ip'
-
-Lookup IP using GreyNoise's free community endpoint
-
-Type: **investigate** \
-Read only: **True**
-
-The Community API provides community users with a free tool to query IPs in the GreyNoise dataset and retrieve a subset of the full IP context data returned by the IP Lookup API.
-
-#### Action Parameters
-
-PARAMETER | REQUIRED | DESCRIPTION | TYPE | CONTAINS
---------- | -------- | ----------- | ---- | --------
-**ip** | required | IP to query | string | `ip` |
-
-#### Action Output
-
-DATA PATH | TYPE | CONTAINS | EXAMPLE VALUES
---------- | ---- | -------- | --------------
-action_result.parameter.ip | string | `ip` | 71.6.135.131 |
-action_result.data.\*.classification | string | | benign malicious |
-action_result.data.\*.last_seen | string | | 2020-01-01 |
-action_result.data.\*.link | string | `url` | https://viz.greynoise.io/riot/1.2.3.4 |
-action_result.data.\*.message | string | | Success You have hit your daily rate limit of 100 requests per day. Please create a free account or upgrade your plan at https://greynoise.io/pricing. |
-action_result.data.\*.name | string | | Cloudflare |
-action_result.status | string | | success failed |
-action_result.message | string | | |
-action_result.summary | string | | |
-summary.total_objects | numeric | | 1 |
-summary.total_objects_successful | numeric | | 1 |
-
-## action: 'similar noise ips'
+## action: 'noise ip timeline'
 
 Lookup Similar internet scanner IP using GreyNoise's IP Similarity tool
```

Comment:
I believe this is the wrong description.

Reply:
Updated the description in the JSON file, so it will reflect here as well.

Reply:
We run the pre-commit checks locally, which automatically generate the README file.